viewing-data.html

```diff
@@ -456,12 +481,10 @@
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
-<li><p class="first">Metadata associated with any level from screens to wells can contain annotations such as gene or siRNA identifiers.</p>
+<li><p class="first">Metadata associated with any level from screens to wells to images can contain annotations such as gene or siRNA identifiers.</p>
```

Comment:
It's the Images we're annotating in the clients. We don't really expose 'fields'.